src/representations/calculator_spherical_invariants.hh

```diff
@@ -38,6 +38,8 @@
 #include "rascal_utility.hh"
 #include "math/math_utils.hh"
 
+#include "wigxjpf.h"
+
```

Comment:
this is an external library so I prefer to import it through the math_interface header.

Reply:
Except that this means the header is included in all compilation units, thus driving the compile time up. But you are right that this should use `#include <wigxjpf.h>` since it is an external library.

Reply:
the code it refers to is only used in this header so I though even if it is included it would be ignored. So at most it increases the size of the file to be compiled but not the compilation time. Is my impression wrong ?

Reply:
Having more code to compile increase compilation time, since the files need to be parsed even if they are not used, which takes up time and compiler memory.